tests/pytests/unit/modules/test_network.py

```diff
@@ -1,14 +1,31 @@
+import logging
+import os.path
+import shutil
+import socket
 import threading
 
 import pytest
 
+import salt.config
 import salt.modules.network as networkmod
-from tests.support.mock import patch
+import salt.utils.path
+from salt._compat import ipaddress
+from salt.exceptions import CommandExecutionError
+from tests.support.mock import MagicMock, mock_open, patch
+
+log = logging.getLogger(__name__)
 
 
 @pytest.fixture
 def configure_loader_modules():
-    return {networkmod: {}}
+    ## return {networkmod: {}}
+    opts = salt.config.DEFAULT_MINION_OPTS.copy()
+    utils = salt.loader.utils(
+        opts, whitelist=["network", "path", "platform", "stringutils"]
+    )
+    return {
+        networkmod: {"__utils__": utils},
+    }
```

Comment:
```suggestion
    ## return {networkmod: {}}
    utils = salt.loader.utils(
        minion_opts, whitelist=["network", "path", "platform", "stringutils"]
    )
    return {
        networkmod: {"__utils__": utils},
    }
```
Add `minion_opts` to `configure_loader_modules()` as an argument